packages/berry-core/sources/VirtualFetcher.ts

```diff
@@ -41,39 +40,37 @@ export class VirtualFetcher implements Fetcher {
   }
 
   getLocatorFilename(locator: Locator) {
-    return structUtils.slugifyLocator(locator);
+    return structUtils.slugifyLocator(locator) as PortablePath
```

Comment:
Not entirely sure, though - I saw that in other places (like `readdir`) we are in the same case and it might not make sense there to return a raw `string`. Your call.

Reply:
> create `FileName` type

Nevermind 😅